### ISSUE TITLE
[Bug] parsing Error

### ISSUE THREAD

**Issue by Ram-ba-Zam-ba** [the-marenga/sf-scrapbook-helper#88]
i tryed your helper yesterday the first time and it worked great. today i could not log into character with the helper, always got a parsing error, but only with this character, not all

OS: Win11
Version: 0.2.2

[helper.log](https://github.com/user-attachments/files/16082299/helper.log)
<img width="769" alt="Screenshot 2024-07-03 122450" src="https://github.com/the-marenga/sf-scrapbook-helper/assets/174604513/a11da8aa-a780-404c-bfbc-75430e070821">

**Comment by the-marenga** [comment]
yeah, they released an update (22.000) to their servers, which broke a few things. I am working on fixing it

**Comment by the-marenga** [comment]
Should be fixed in the newest release. The S&F update is pretty big, so there may be more things wrong, that I have not found yet. If you see any other errors, feel free to open a new issue 😄

**Comment by Ram-ba-Zam-ba** [comment]
that was quick, big thanks. its working again. eu13 here i come
